spectrum/templates/base.html

```diff
@@ -16,7 +16,11 @@
     <link rel="stylesheet" href="{% static 'info/css/global.css' %}">
     <link rel="stylesheet" href="{% static 'info/css/events.css' %}">
     <script src="https://kit.fontawesome.com/0b4fbfd8b7.js" crossorigin="anonymous"></script>
-    {% block stylesheets %}{% endblock %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>           
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.7.1/js/bootstrap-datepicker.min.js"></script>
```

Comment:
This _should_ already be included with django.

Reply:
It's probably not the solution we're looking for then, because jquery _is_ already present in the code.